### PR TITLE
Search implementation

### PR DESCRIPTION
**Pull request by svetlanapelevina**

### CODE REVIEW

File: forms.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ class DogForm(FlaskForm):
     wise_nose_id = StringField('Wise Nose ID', validators=[Length(min=0, max=100)])
     trainer = SelectField('Trainer name', validators=[DataRequired()], coerce=int)
     photo = FileField('Dog photo', validators=[FileAllowed(['png', 'jpg', 'jpeg', 'gif'], 'Images only!')])
+    search = StringField()
```
Comment (svetlanapelevina) [inline forms.py]: that's old version, we dont need it
Sorry, I am too lazy to create a new pull request only for one line ^^

File: templates/exportmembers.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	<label for="person_name">Member name
+		<input type="text" id="person_name" placeholder="Member name" class="form-control form-control-lg" value={{search_data.name}}>
+	</label>
+	<label for="role">Role
+<!--		<select id="role" class="form-control form-control-lg">-->
+<!--			<option hidden selected value={{search_data.role}}>{{search_data.role}}</option>-->
+<!--			{% for trainer in trainers %}-->
+<!--				<option>{{ trainer.name }}</option>-->
+<!--			{% endfor %}-->
+<!--		</select>-->
```
Comment (JonathanMbt) [inline templates/exportmembers.html]:
```suggestion
<select id="role" class="form-control form-control-lg">
      <option {%if search_data.role == "1"%}selected{%endif%} value="1">Trainer</option>
      <option {%if search_data.role == "2"%}selected{%endif%}value="2">Supervisor</option>
      <option {%if search_data.role == "3"%}selected{%endif%} value="3">Trainer/Supervisor</option>
</select>
```

File: templates/members.html [hunk 3]
```diff
@@ -5,6 +5,26 @@
...
+<label for="person_name">Member name
+    <input type="text" id="person_name" placeholder="Member name" class="form-control form-control-lg" value={{search_data.name}}>
+</label>
+<label for="role">Role
+<!--		<select id="role" class="form-control form-control-lg">-->
+<!--			<option hidden selected value={{search_data.role}}>{{search_data.role}}</option>-->
+<!--			{% for trainer in trainers %}-->
+<!--				<option>{{ trainer.name }}</option>-->
+<!--			{% endfor %}-->
+<!--		</select>-->
```
Comment (JonathanMbt) [inline templates/members.html]:
```suggestion
<select id="role" class="form-control form-control-lg">
      <option {%if search_data.role == "1"%}selected{%endif%} value="1">Trainer</option>
      <option {%if search_data.role == "2"%}selected{%endif%}value="2">Supervisor</option>
      <option {%if search_data.role == "3"%}selected{%endif%}
      value="3">Trainer/Supervisor</option>
</select>
```

**Review by JonathanMbt — APPROVED** [review]

File: templates/exportmembers.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	<label for="person_name">Member name
+		<input type="text" id="person_name" placeholder="Member name" class="form-control form-control-lg" value={{search_data.name}}>
+	</label>
+	<label for="role">Role
+<select id="role" class="form-control form-control-lg">
+      <option {%if search_data.role == "1"%}selected{%endif%} value="1">Trainer</option>
+      <option {%if search_data.role == "2"%}selected{%endif%}value="2">Supervisor</option>
+      <option {%if search_data.role == "3"%}selected{%endif%} value="3">Trainer/Supervisor</option>
+</select>
+    	<input type="text" id="role" placeholder="Role" class="form-control form-control-lg" value={{search_data.role}}>
```
Comment (JonathanMbt) [inline templates/exportmembers.html]:
```suggestion

```

File: templates/members.html [hunk 5]
```diff
@@ -5,6 +5,26 @@
...
+    <input type="text" id="person_name" placeholder="Member name" class="form-control form-control-lg" value={{search_data.name}}>
+</label>
+<label for="role">Role
+<select id="role" class="form-control form-control-lg">
+      <option {%if search_data.role == "1"%}selected{%endif%} value="1">Trainer</option>
+      <option {%if search_data.role == "2"%}selected{%endif%}value="2">Supervisor</option>
+      <option {%if search_data.role == "3"%}selected{%endif%}
+      value="3">Trainer/Supervisor</option>
+</select>
+    <input type="text" id="role" placeholder="Role" class="form-control form-control-lg" value={{search_data.role}}>
```
Comment (JonathanMbt) [inline templates/members.html]:
```suggestion

```

**Review by JonathanMbt — APPROVED** [review]

File: templates/exportmembers.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+{% block body %}
+
+<div class="container">
+	<label for="person_name">Member name
+		<input type="text" id="person_name" placeholder="Member name" class="form-control form-control-lg" value={{search_data.name}}>
+	</label>
+	<label for="role">Role
+<select id="role" class="form-control form-control-lg">
+      <option {%if search_data.role == "1"%}selected{%endif%} value="1">Trainer</option>
+      <option {%if search_data.role == "2"%}selected{%endif%}value="2">Supervisor</option>
```
Comment (JonathanMbt) [inline templates/exportmembers.html]:
```suggestion
      <option {%if search_data.role == "2"%}selected{%endif%} value="2">Supervisor</option>
```

File: templates/members.html [hunk 7]
```diff
@@ -5,6 +5,25 @@
 {% endblock %}
 
 {% block body %}
+<label for="person_name">Member name
+    <input type="text" id="person_name" placeholder="Member name" class="form-control form-control-lg" value={{search_data.name}}>
+</label>
+<label for="role">Role
+<select id="role" class="form-control form-control-lg">
+      <option {%if search_data.role == "1"%}selected{%endif%} value="1">Trainer</option>
+      <option {%if search_data.role == "2"%}selected{%endif%}value="2">Supervisor</option>
```
Comment (JonathanMbt) [inline templates/members.html]:
```suggestion
      <option {%if search_data.role == "2"%}selected{%endif%} value="2">Supervisor</option>
```

**Review by JonathanMbt — APPROVED** [review]

**Pull request merged**